.github/workflows/ci.yml

```diff
@@ -23,8 +23,6 @@ jobs:
     - name: Install dependencies
       run: |
         npm ci
-        npm run generate-component-install
```

Comment:
nit: I think you can also remove `|` now and just write `run: npm ci`

Reply:
Done